testing/SendLog/SendLog.py

```diff
@@ -0,0 +1,95 @@
+## Hrishi Hiraskar
+## 8 September 2016
+
+import gevent
+import time
+from gevent import monkey
+from gevent.pywsgi import WSGIServer
+from flask import Flask, request, Response, render_template
+
+monkey.patch_all()
+
+app = Flask(__name__)
+
+# Delay time to look for new line (in s)
+LOOK_DELAY = 0.01	
+
+def get_line(file):
+	# Function to get a new line from file
+	line = file.readline()
+	while not line:
```

Comment:
This loop will end as soon as a new line from file is fetched
Consider a case where we have read till the end of file, so we will get NULL if we call read_line()
But we know that new figures may come, or data of some existing figure may come
So we will need to wait till the new line comes, that's what the function does